<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSetting</class>
 <widget class="QDialog" name="SerialSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>341</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelComInfo">
          <property name="text">
           <string>Доступные порты:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="serialPortInfoListBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelSpeed">
            <property name="text">
             <string>Скорость передачи:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="baudRateBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="labelDataBit">
            <property name="text">
             <string>Количество бит данных:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="dataBitsBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Бит четности:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="parityBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Количество стоп бит:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="stopBitsBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Управление потоком:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="flowControlBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="localEchoCheckBox">
      <property name="text">
       <string>Включить эхо</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="redreshButton">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="apllySettButton">
        <property name="text">
         <string>Подключиться</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
